src/sortStudents.ts

```diff
@@ -1,16 +1,50 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade'
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+// eslint-disable-next-line
+export function sortStudents(students: Student[], sortBy: SortType, order: SortOrder): Student[] {
+  const copyStudents: Student[] = JSON.parse(JSON.stringify(students));
```

Comment:
I think this would be enough in this particular case

```suggestion
  const copyStudents: Student[] = [...students];
```